src/main/java/software/amazon/cloudwatchlogs/emf/model/MetricDefinition.java

```diff
@@ -43,18 +47,36 @@ class MetricDefinition {
     @JsonDeserialize(using = UnitDeserializer.class)
     private Unit unit;
 
+    @Getter
+    @Setter
+    @JsonProperty("StorageResolution")
+    @JsonInclude(
+            value = JsonInclude.Include.CUSTOM,
+            valueFilter =
+                    StorageResolutionFilter.class) // Do not serialize when valueFilter is true
+    @JsonSerialize(using = StorageResolutionSerializer.class)
+    public StorageResolution storageResolution;
+
     @JsonIgnore @NonNull @Getter private List<Double> values;
 
     MetricDefinition(String name) {
-        this(name, Unit.NONE, new ArrayList<>());
+        this(name, Unit.NONE, StorageResolution.STANDARD, new ArrayList<>());
     }
 
     MetricDefinition(String name, double value) {
-        this(name, Unit.NONE, value);
+        this(name, Unit.NONE, StorageResolution.STANDARD, value);
     }
 
     MetricDefinition(String name, Unit unit, double value) {
-        this(name, unit, new ArrayList<>(Arrays.asList(value)));
+        this(name, unit, StorageResolution.STANDARD, new ArrayList<>(Arrays.asList(value)));
+    }
+
+    MetricDefinition(String name, StorageResolution storageResolution, double value) {
```

Comment:
No test coverage

Reply:
Can we change the order of this parameter such as name, value, storageresolution.

Reply:
> No test coverage

Thanks for pointing this out. Added tests [here](https://github.com/awslabs/aws-embedded-metrics-java/pull/129/files#diff-c3cf6b434742851943b50c2f968e3b3d211f3fa84247c56094155b01aafdd113R34).